packages/suite/src/support/messages.ts

```diff
@@ -228,7 +228,7 @@ export default defineMessages({
     },
     TR_EXCHANGE_DEX_OFFER_FEE_INFO: {
         defaultMessage:
-            'The fees to perform this swap are estimated at {symbol} {approvalFee} ({approvalFeeFiat}) for approval (if required) and {symbol} {swapFee} ({swapFeeFiat}) for the swap.',
+            'The fees to perform this swap are estimated at {approvalFee} ({approvalFeeFiat}) for approval (if required) and {swapFee} ({swapFeeFiat}) for the swap.',
         id: 'TR_EXCHANGE_DEX_OFFER_FEE_INFO',
```

Comment:
We have to make sure that the modified messages will be propagated to other languages. I am not sure what is the right procedure for that but we have to take care of that.

Reply:
Sure, it's in my TODO list.